playbooks/scale-up.yaml

```diff
@@ -0,0 +1,43 @@
+---
+# Set new number of nodes
+- hosts: masters
+  tasks:
+  - name: Get number of nodes
+    shell: oc get nodes | grep app-node -c
+    register: oc_old_result
+  - name: Set new number of nodes
+    set_fact:
+      # Number of nodes in the new configuration (incremented by 1 by default)
+      nodes: "{{ oc_old_result.stdout | int + 1 }}"
+  - name: Check that new value is higher
```

Comment:
Can we also verify that the `oc get nodes` value matches the existing `openstack_num_node`?

Basically making sure that whatever's set in the vars actually represents the reality.